hazelcast/src/test/java/com/hazelcast/flakeidgen/impl/FlakeIdGenerator_MemberBackpressureTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.flakeidgen.impl;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.config.FlakeIdGeneratorConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.flakeidgen.FlakeIdGenerator;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.TestHazelcastInstanceFactory;
+import org.junit.After;
+import org.junit.Test;
+
+import static java.util.concurrent.TimeUnit.NANOSECONDS;
+import static org.junit.Assert.assertTrue;
+
+public class FlakeIdGenerator_MemberBackpressureTest extends HazelcastTestSupport {
```

Comment:
Missing `@RunWith` and `@Category` here (I guess this should run as `SlowTest` with the serial runner so its execution is as isolated as possible to avoid false failures)

Reply:
fixed